backend/webapp/src/main/java/org/bonitasoft/web/designer/controller/utils/MimeType.java

```diff
@@ -14,10 +14,21 @@
  */
 package org.bonitasoft.web.designer.controller.utils;
 
+import org.springframework.util.MimeTypeUtils;
+
 public enum MimeType {
 
     APPLICATION_ZIP("application/zip", "application/x-zip-compressed", "application/x-zip"),
-    APPLICATION_OCTETSTREAM("application/octet-stream");
+    APPLICATION_OCTETSTREAM("application/octet-stream"),
+    WEB_RESOURCES(
+            MimeTypeUtils.APPLICATION_JSON_VALUE,
+            MimeTypeUtils.APPLICATION_XML_VALUE,
+            MimeTypeUtils.IMAGE_GIF_VALUE,
+            MimeTypeUtils.IMAGE_JPEG_VALUE,
+            MimeTypeUtils.IMAGE_PNG_VALUE,
+            MimeTypeUtils.TEXT_HTML_VALUE,
+            MimeTypeUtils.TEXT_XML_VALUE,
```

Comment:
I'm not sure we need these Mimetypes (kson, xml, html, etc....).
For me, web resources are images, css and js. 
Furthermore, as we seen earlier in pages import, this is dangerous to restrict import from content-type since we don't know exactly what the browser will send